js/app/fishwatcher.js

```diff
@@ -212,9 +212,17 @@ class FishWatcher {
     if (_(fish.predators).size() > 0) {
       var overallPredRange = null;
       var predatorsAlwaysAvailable = true;
+      // This key value is in seconds, we will need to convert it to Eorzean time first
+      var intuitionLength = fish.intuitionLength;
+      // If no key was defined, default to 1 Eorzean hour duration
+      // Keeping precision down to seconds here because of non-aligning intuition durations
+      // (e.g. 100 Earth seconds)
+      if (intuitionLength !== undefined) {
```

Comment:
Ah  nuts, it's actually generating `null` for this when it dumps the thing to JSON to create data.js.
Add intuitionLength !== null to the if statement as well (similar pattern to in fish.js for gig and dataMissing)
```suggestion
      if (intuitionLength !== null && intuitionLength !== undefined) {
```